mlrun/model_monitoring/stream_processing_fs.py

```diff
@@ -719,6 +726,46 @@ def do(self, full_event):
         storey_event = storey.Event(body=events, key=endpoint_id)
         return storey_event
 
+    def _update_request_timestamps(
+        self, endpoint_id: str, timestamp: str
+    ) -> datetime.datetime:
+        """Update model endpoint first and last request timestamps. In addition, this function validates that the
```

Comment:
i think it's better for function to do one thing. in this case - compare last_event time to current_event time.
in other words, maintenance of first_request/last_request should be outside